src/Schema/Domain/Content/Worker/FieldDefinition/AddFieldDefinitionToDomainContentMutation.php

```diff
@@ -95,12 +101,21 @@ protected function getFieldDefinitionField(FieldDefinition $fieldDefinition): st
         return $this->getNameHelper()->fieldDefinitionField($fieldDefinition);
     }
 
-    private function getFieldDefinitionToGraphQLType(FieldDefinition $fieldDefinition, $operation): string
+    private function fieldType(array $args, $operation): string
     {
+        $fieldDefinition = $args['FieldDefinition'];
+        $contentType = $args['ContentType'];
```

Comment:
Since you've already extracted this in `work` method, maybe here you could pass them directly instead of reading `$args` again?
I mean passing:
`ContentType $contentType, FieldDefinition $fieldDefinition` as arguments to this method.

Reply:
You're right, good catch.